companyweb_base/wizards/credential_wizard.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 ACSONE SA/NV
+from odoo import fields, models
+
+
+class CompanyWebCredentialWizard(models.TransientModel):
+
+    _name = "companyweb_base.credential_wizard"
+    _description = "Ask for CompanyWeb login & password"
+
+    cweb_login = fields.Char("Company Web Login", required=True)
+    cweb_password = fields.Char("Company Web Password", required=True)
+
+    def save_cweb_login_pwd(self):
+        """
+        save information given in the form to the logged-in user
```

Comment:
You may want to have a look at https://www.python.org/dev/peps/pep-0257/ for docstring style.